.changeset/tricky-toes-boil.md

```diff
@@ -0,0 +1,7 @@
+---
+'@rdfjs-elements/editor-base': patch
+'@rdfjs-elements/rdf-editor': patch
+'@rdfjs-elements/rdf-snippet': patch
+---
+
+Remove the default prefixes `rdf, rdfs, xsd` (closes #95)
```

Comment:
@cristianvasquez I'm not sure whether this is a breaking change. Not in technical terms, since any usage will continue to work, but the behaviour will change slightly